src/main/kotlin/no/nav/familie/ef/mottak/task/ArkiverSøknadTask.kt

```diff
@@ -30,7 +31,7 @@ class ArkiverSøknadTask(private val arkiveringService: ArkiveringService,
         } else {
             Task(TaskType(TYPE).nesteFallbackTask(), task.payload, task.metadata)
         }
-
+        task.metadata["eventId"] = UUID.randomUUID()
```

Comment:
Er denne egentlige allerede i task.metadata fra `TaskProsesseringService.kt` ?

Reply:
Den finnes allerede i `TaskProsesseringService` for å generere egen UUID ved opprettelse av `SendSøknadMottattTilDittNavTask. Har skjønt det slik at metadata blir kopiert til neste task, så her prøver jeg derfor å overskrive eksisterende eventId med ny UUID før opprettelse av `SendDokumentasjonsbehovMeldingTilDittNavTask`.

Reply:
Ah 👍 

Her skriver vi også over eventId i arkiverSøknadTask som settes i TaskProsesseringService
Den er ikke helt immutable

Lurer på om men kanskje burde kopiere properties og overskreve den? Litt usikker
```
val sendMeldingTilDittNavTask =
            Task(SendDokumentasjonsbehovMeldingTilDittNavTask.TYPE,
                task.payload,
                Properties(task.metadata).apply {
                    this["eventId"] = UUID.randomUUID()
                })
```

Reply:
Hm, tenker uansett at det er ryddigere å kopiere ut og deretter overskrive slik som i forslaget ditt, så jeg har endret det :)